src/components/layouts/footer/Footer.js

```diff
@@ -17,4 +18,10 @@ Footer.propTypes = {
   appVersion: PropTypes.string.isRequired,
 };
 
-export default React.memo(Footer);
+function mapStateToProps(state) {
```

Comment:
Looks like a change not related to router. Let's move to earlier commit where footer was just introduced